tests/unit/swagger/definitionsGeneration/metadata.spec.ts

```diff
@@ -14,6 +14,22 @@ describe('Metadata generation', () => {
     });
   });
 
+  describe('DynamicControllerGenerator', () => {
+    it('should should throw \'globs found 0 controllers.\'', () => {
+      expect(() => {
+        // Non existing controllers folder to get 0 controllers found error
+        const NON_CONTROLLER_EXISTS_GLOB = './tests/unit/swagger';
+        new MetadataGenerator('./tests/fixtures/express-dynamic-controllers/server.ts', undefined, [], [NON_CONTROLLER_EXISTS_GLOB] ).Generate();
+      }).to.throw(/globs found 0 controllers./);
+    });
+
+    it('should generate one controller', () => {
+      expect(metadata.controllers.length).to.equal(1);
```

Comment:
It was probably just a slip up, but this second unit test isn’t actually testing the controllers glob feature because it uses the metadata variable at the top of the file. You’d need to start the test off with:
```
const metadataForControllerGlobs = new MetadataGenerator( /* same settings you did in the last test */ );

expect(metadataForControllerGlobs.controllers[0]...
```